### DISCUSSION TITLE
Is it possible to change rustdesk self-hosting server's info via command line?

### DISCUSSION THREAD

**Discussion by Seeker98** [rustdesk/rustdesk#8940] [Q&A]
Hello, currently i cant connect to my server using rustdesk because the self-hosting server is invalid, and unchecking "force self-hosting" also does not solve. but at present i can connect via remote ssh. I want to ask if it is possible to change info of self-hosting server via commandline (eg. vim some config file?) and restart rustdesk service. if it is possible, how should i operate or refer to which document, thanks.

**Comment by rustdesk** [comment]
#763, go to [discord](https://discord.gg/nDceKgxnkV) please, usually nobody will help you with self-hosting here.
